static/js/controllers/inbox.js

```diff
@@ -661,28 +662,32 @@ var feedback = require('../modules/feedback'),
         $rootScope.$broadcast.apply($rootScope, arguments);
       };
 
-      var docToDeleteId;
-
-      $scope.deleteDoc = function(id) {
-        $('#delete-confirm').modal('show');
-        docToDeleteId = id;
+      // TODO promisify DeleteDoc
```

Comment:
Don't commit TODOs - raise an issue instead. The only time I see TODOs is when I'm busy doing something else, whereas I see issues when I'm looking for something to work on.